operator/builtin/output/count/count.go

```diff
@@ -0,0 +1,149 @@
+package counter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"math/big"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/observiq/stanza/entry"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/operator/helper"
+)
+
+type CountOutputConfig struct {
+	helper.OutputConfig `yaml:",inline"`
+	Path                string          `json:"path,omitempty" yaml:"path"`
+	Duration            helper.Duration `json:"duration,omitempty" yaml:"duration,omitempty"`
+}
+
+var defaultCounterDuration = helper.NewDuration(1 * time.Minute)
+
+func init() {
+	operator.Register("count_output", func() operator.Builder { return NewCounterOutputConfig("") })
+}
+
+func NewCounterOutputConfig(operatorID string) *CountOutputConfig {
+	return &CountOutputConfig{
+		OutputConfig: helper.NewOutputConfig(operatorID, "count_output"),
+		Duration:     defaultCounterDuration,
+	}
+}
+
+func (c CountOutputConfig) Build(bc operator.BuildContext) ([]operator.Operator, error) {
+	outputOperator, err := c.OutputConfig.Build(bc)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	logChan := make(chan struct{}, 1)
+	counterOperator := &CountOutput{
+		OutputOperator: outputOperator,
+		ctx:            ctx,
+		cancel:         cancel,
+		interval:       c.Duration.Raw(),
+		path:           c.Path,
+		numEntries:     big.NewInt(0),
+		logChan:        logChan,
+		wg:             sync.WaitGroup{},
+	}
+
+	return []operator.Operator{counterOperator}, nil
+}
+
+type CountOutput struct {
+	helper.OutputOperator
+	ctx      context.Context
+	interval time.Duration
+	start    time.Time
+	logChan  chan struct{}
+	encoder  *json.Encoder
+	path     string
+	wg       sync.WaitGroup
+	cancel   context.CancelFunc
+
+	numEntries *big.Int
```

Comment:
Curious why we didn't use a `uint64`? A `big.Int` type seems like an over optimization unless we think we could overflow a `uint64`.